README.md

```diff
@@ -133,7 +116,97 @@ include the `restPort` configuration to tell the exporter which port to use.
 * [Chart-based sample](samples/kubernetes/end2end): This is an end-to-end sample that shows you the steps to set up monitoring for WebLogic domains using Prometheus and Grafana. Prometheus and Grafana are installed with Helm charts.
 
 
+# Web Application
+
+## Setting the configuration
+
+The Web Application has a main landing page which displays the current [configuration](#configuration) and allows
+a user to change it, either by uploading a replacement or an addition to the queries specified with the current one.
+Metrics will then be available from `<application-root>/metrics`.
+
+
+## Downloading the release
+
+You can find all the releases on the [Releases page](https://github.com/oracle/weblogic-monitoring-exporter/releases).
+
+To download the web application `wls-exporter.war` and put your configuration file into the WAR, download the `getXXX.sh` script, which is provided with each release and also can be downloaded from the Releases page, and then run:
+
+```
+bash getXXX.sh <your-config-file>
+```
+
+## Building from source
+
+Use `mvn install` to build the web application. This will create `wls-exporter-<version>`, where _version_
+is the Maven-assigned version number.
+
+Adding `-Dconfiguration=<some-config-file>` will insert the specified configuration as its default and remove
+the version number to simplify deployment to WebLogic Server.
+
+# Sidecar
+
+## Build and run
+
+The sidecar implementation must be built with JDK11 or later. Building the project with JDK8 will skip the sidecar module.
+
```

Comment:
I am not understanding where the sidecar build command is. 
You state that you need to build the sidecar implementation with JDK11 or later. Building the project with JDK8 will skip the sidecar module. 
Then, the next line gives a command for "after building."